src/main/java/storagesystem/model/ReservationHandler.java

```diff
@@ -0,0 +1,149 @@
+package storagesystem.model;
+
+import org.joda.time.Interval;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A handler taking care of all reservations for one organisation. Methods for creating and retrieving reservations
+ */
+
+public class ReservationHandler {
+
+
+    private List<IReservation> reservations;
+
+
+    public ReservationHandler(int nextReservationID, List<IReservation> reservations) {
+        Reservation.setNxtResId(nextReservationID);
+        this.reservations = reservations;
+    }
```

Comment:
Keep track of nextReservationID in this class instead? Or get it from the latest created reservation somehow?

Doesn't feel correct that nextReservationID should be a param for ReservationHandler's constructor.

Shouldn't the list of reservations be empty when creating an instance of the class aswell? And then maybe a function which has capability of adding a list of reservations to this ReservationHandler's list?

Reply:
I do think nextReservationID should stay in reservation, but agree with the rest

Reply:
I do think nextReservationID should stay in reservation, but agree with the rest